### PR TITLE
adding soft deletion for comments and posts

### PR DESCRIPTION
**Pull request by marinaRomanchuk**

### CODE REVIEW

**Comment by revimi** [comment]
Looks good! But we should also filter out the deleted stuff and don't return it in our endpoints' responses

File: comments/views.py [hunk 1]
```diff
@@ -9,12 +9,12 @@
 
 
 class CommentViewSet(viewsets.ModelViewSet):
-    queryset = Comment.objects.all()
+    queryset = Comment.objects.filter(is_deleted=False)
```
Comment (revimi) [inline comments/views.py]: It's better to override `objects` to be 100% sure we don't forget to filter out deleted objects in future 
https://stackoverflow.com/questions/4762524/is-it-possible-to-override-objects-on-a-django-model
Same comment for posts

**Comment by marinaRomanchuk** [comment]
That's true, filtering was added

**Comment by revimi** [comment]
Please follow this link and create custom managers for posts and comments
https://stackoverflow.com/questions/4762524/is-it-possible-to-override-objects-on-a-django-model

File: comments/models.py [hunk 2]
```diff
@@ -1,11 +1,19 @@
 from django.db import models
 
 
+class CommentManager(models.Manager):
+    def get_queryset(self):
+        return super(CommentManager, self).get_queryset().filter(is_deleted=False)
```
Comment (revimi) [inline comments/models.py]:
```suggestion
class CommentManager(models.Manager):

    def get_queryset(self):
        return super().get_queryset().filter(is_deleted=False)
```

File: posts/models.py [hunk 3]
```diff
@@ -1,11 +1,19 @@
 from django.db import models
 
 
+class PostManager(models.Manager):
+    def get_queryset(self):
+        return super(PostManager, self).get_queryset().filter(is_deleted=False)
```
Comment (revimi) [inline posts/models.py]:
```suggestion
class PostManager(models.Manager):

    def get_queryset(self):
        return super().get_queryset().filter(is_deleted=False)
```

**Review by revimi — APPROVED** [review]
Great! Approving with minor "nice to have" suggestions

File: comments/models.py (hunk 2 above)
Comment (marinaRomanchuk) [reply comments/models.py]: Done

File: posts/models.py (hunk 3 above)
Comment (marinaRomanchuk) [reply posts/models.py]: Done

**Pull request merged**